File: containerSetup/submissions/2/Main.cpp
#include <iostream>
using namespace std;

int main()
{
    long long n;
    cin >> n;

    long long original = n;
    long long reversed = 0;

    while (n > 0)
    {
        reversed = reversed * 10 + (n % 10);
        n /= 10;
    }

    cout << (original == reversed ? "true" : "false") << endl;

    return 0;
}

File: containerSetup/submissions/2/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        long n = sc.nextLong();

        long original = n;
        long reversed = 0;

        while (n > 0) {
            reversed = reversed * 10 + (n % 10);
            n /= 10;
        }

        System.out.println(original == reversed ? "true" : "false");
    }
}